OasisDEX/oasis-borrow
# 🧭 Discover filters refinements and transformators

# 🧭 Discover filters refinements and transformators
  
## Changes 👷‍♀️

![image](https://user-images.githubusercontent.com/16230404/197789110-da8a7206-8918-4ee2-b8c1-eb46ecdf1201.png)
![image](https://user-images.githubusercontent.com/16230404/197789204-27c7edd1-fcff-4ae1-837e-15413dff42ea.png)

- Created more universal methods for filtering discover database,
- added better error handling,
- transformed API data from database format to Discover table favored format.
  
## How to test 🧪

Endpoints still aren't connected anywhere on the app, has to be tested directly in the browser. For more info how to test see seeding script from this PR: https://github.com/OasisDEX/oasis-borrow/pull/1590
